runner/job_manager.py

```diff
@@ -413,3 +419,12 @@ def delete_resources(self, batch_service_client: batch.BatchExtensionsClient,
             logger.info(
                 "Job: {}. did not complete successfully, Container {} was not deleted.".format(
                     self.job_id, self.storage_info.output_container))
+
+    def update_params_with_values_from_keyvault(self, parameters, keyvault_client_with_url):
+        for parameter, value in parameters.items():
+            stringValue = str(value)[11:-2] #substring the inner value from format {"value": "<value-we-want>"}
```

Comment:
>stringValue [](start = 12, length = 11)

string_value... also may want a different name

Reply:
Thanks - inner_value is clearer